web/api/v1/api.go

```diff
@@ -381,6 +383,7 @@ func (api *API) Register(r *route.Router) {
 	r.Get("/targets", wrap(api.targets))
 	r.Get("/targets/metadata", wrap(api.targetMetadata))
 	r.Get("/alertmanagers", wrapAgent(api.alertmanagers))
+	r.Get("/servicediscovery", wrapAgent(api.servicediscovery))
```

Comment:
why do we need wrapAgent? this should be available with the agent.

Reply:
👌，thanks